File: Restaurant-information-system/src/main/java/com/galaxy/Restaurantinformationsystem/Entity/ReviewEntity.java
package com.galaxy.Restaurantinformationsystem.Entity;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity
public class ReviewEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long RPK;
    private String title;
    private String content;
    private String image;
    @ManyToOne
    private StoreEntity storeEntity;
    @ManyToOne
    private UserEntity userEntity;
}

File: src/main/java/com/galaxy/restaurant/information/system/entity/ReviewEntity.kt
package com.galaxy.restaurant.information.system.entity

import jakarta.persistence.*

@Entity
class ReviewEntity(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long = 0,
    var title: String,
    var content: String,
    var image: String?,
    var score: Int,
    @ManyToOne
    val store: StoreEntity,
    @ManyToOne
    val author: UserEntity
) {
    override fun toString(): String {
        return "ReviewEntity(id=$id, title='$title', content='$content', image=$image, score=$score, store=$store, author=$author)"
    }
}
